app/controllers/api/v2/cautionary_alerts_controller.rb

```diff
@@ -4,7 +4,7 @@ module Api
   module V2
     class CautionaryAlertsController < ApplicationController
       def index
-        @property = CautionaryAlert.for_property_reference(params[:property_id])
+        @property = BuildCautionaryAlertsService.new(params[:property_id]).perform
```

Comment:
Might be worth a quick rename of `@property` to avoid confusion  ... `@cautionary_alerts` ?

Reply:
How about moving the `BuildCautionaryAlertsService.new(params[:property_id]).perform` into `CautionaryAlerts` class?

Reply:
Hey how exactly did you mean by this @EGiataganas https://github.com/LBHackney-IT/repairs-api/pull/20#discussion_r540955599 ?